packages/engine-browser/src/query-builder.test.ts

```
@@ -0,0 +1,2300 @@
+          { columnName: "created_at", operator: ">=", value: testDate },
+        ]);
+        const sql = await qb.sql();
+
+        expect(sql).toBe(
+          'SELECT * FROM "df_test_df_id" WHERE "created_at" >= \'2024-01-15T10:30:00.000Z\'',
+        );
+      });
+
+      it("should handle bigint values", async () => {
+        const qb = createTestQueryBuilder(mockDataFrame, mockConn).filter([
+          { columnName: "id", operator: "=", value: BigInt(9007199254740993) },
```

Biome (2.1.2)

[error] This number literal will lose precision at runtime.

The value at runtime will be 9007199254740992

(lint/correctness/noPrecisionLoss)